Develop

carlwang1995 opened the pull request:
王維耀
http://54.243.157.143:8000/
http://54.243.157.143:8000/attraction/10

carlwang1995 committed:
Fix 'prevent repeat loading' feture, and some CSS code. 12/06/24

carlwang1995 committed:
Add database connection pool.

carlwang1995 committed:
Complete week 3 task. --version 1

cwpeng reviewed (changes requested):
- There is a separting line between navigation bar and attraction images in desktop layout.
- There is no spacing between navigation bar and attraction images in mobile layout.
![54 243 157 143_8000_attraction_10](https://github.com/carlwang1995/tdt/assets/6104683/1f02eeb6-1531-4cf2-b1e3-cc43daf2b505)

carlwang1995 committed:
Fixed problems, have done as below:
1. Add separting line between navigation bar and mian content.
2. Removed spacing between navigation bar and mian content in mobile layout.

carlwang1995 commented:
Problems are fixed , have done as below:
1. Add separting line between navigation bar and main content.
2. Removed spacing between navigation bar and main content in mobile layout.

cwpeng reviewed (approved):
Good. Done.
